test/functionalTest/cases/0876_attribute_dates/attribute_dates.test

```diff
@@ -0,0 +1,212 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
It would be interesting to include also a .test for the subscription/notification case